web-registry/src/pages/ProjectReview/ProjectReview.util.tsx

```diff
@@ -27,41 +29,53 @@ export const getOnChainProjectId = (
 };
 
 export const getJurisdiction = async (
-  metadata: Partial<ProjectMetadataLD> | Partial<VCSProjectMetadataLD>,
+  metadata: Partial<ProjectMetadataLDUnion>,
 ): Promise<string | undefined> => {
   if (!mapboxToken) return Promise.reject('Missing map API token');
   let isoString;
   const location = metadata?.['schema:location'];
-  if (!location?.context || !location?.place_name) {
+  if (!location?.['geojson:context'] && !location?.['geojson:place_name']) {
     return Promise.reject('Please select a location for this project.');
   }
-  const context: GeocodeFeature[] = location?.context || [];
+  const context: any[] = location?.['geojson:context'] || [];
   let countryKey = '';
   let stateProvince = '';
   let postalCode = '';
   context.forEach(ctx => {
-    if (ctx.id.includes('country')) {
-      countryKey = getCountryKey(ctx.text);
+    if (ctx?.['geojson:id'].includes('country')) {
```

Comment:
These shape changes are caused by the `jsonLdCompact` method, which standardizes the data shape.